packages/react/src/SelectPanel/SelectPanel.module.css

```diff
@@ -140,7 +140,6 @@
   display: none;
   align-items: center;
   padding: var(--base-size-8);
-  gap: var(--stack-gap-condensed);
```

Comment:
@hectahertz tagging you since I know you worked on a lot of these, any idea why/if this is needed or are we safe to remove? 👀

Reply:
I think it's a leftover from a previous implementation; if the snapshots look fine I can't think of any issues with removing it!